python/astshim/test.py

```diff
@@ -131,3 +132,66 @@ def checkBasicSimplify(self, amap):
             self.assertEqual(mb.getNout(), cmp3.getNout())
             self.assertEqual(cmp3.getNin(), cmp3simp.getNin())
             self.assertEqual(cmp3.getNout(), cmp3simp.getNout())
+
+
+def makeCoeffs(nIn, nOut):
```

Comment:
The use case is the one you suggested -- avoiding code duplication with Mapping-based tests in `afw`. But I can add PolyMap to the name.